src/searches/useCategorySearch.ts

```diff
@@ -14,6 +14,14 @@ export const searchCategories = gql`
         node {
           id
           name
+          ancestors(first: $first) {
```

Comment:
question: what in case you will have more categories?

Reply:
I wouldn't worry about it now, it's a first iteration. We pass 10 here so it should be fine for most cases. If there are 10+ categories then the direct parents will not appear.